images/openwisp_openvpn/revokelist.sh

```
@@ -4,7 +4,14 @@
 # update CRL periodically.
 cd /
 source /utils.sh
+source /openvpn_utils.sh
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'source' in place of '.' is undefined.

(SC3046)